modules/core/templates/error.twig

```diff
@@ -0,0 +1,25 @@
+{% set pagetitle = 'An error has occurred'|trans %}
+{% extends "base.twig" %}
+
+{% block content %}
+    <h2>{{ pagetitle }}</h2>
+
+    {% if code is same as ('IDENTIFICATION_FAILURE') %}
+    <p>{{ 'The identification procesd has failed.' |trans }}</p>
+    {% elseif code is same as ('AUTHENTICATION_FAILURE') %}
+    <p>{{ 'The authentication procesd has failed.' |trans }}</p>
+    {% elseif code is same as ('AUTHORIZATION_FAILURE') %}
+    <p>{{ 'The authorization procesd has failed.' |trans }}</p>
+    {% else %}
+    <p>{{ 'There was an issue while signing you in.' |trans }}</p>
+    {% endif %}
+
+    {% if is defined ts or is defined rp or is defined tid or is defined ctx %}
```

Comment:
Should this not be
```
{% if ts is defined or rp is defined or tid is defined or ctx is defined %}
```

Reply:
Yes, you are right.. I had fixed this locally, but I failed to upload it.